### PR TITLE
Adds a new periodic Azure Red Hat OpenShift job

### PR DESCRIPTION
**Pull request by m1kola**
The job will be validating the latest marketplace ARO VM image

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 1]
```diff
@@ -187,6 +187,29 @@ periodics:
...
+  decorate: true
+  interval: 24h
+  extra_refs:
+  - org: openshift
+    repo: openshift-azure
+    base_ref: master
+  name: periodic-ci-azure-vmimage-validate-osa311-latest
+  spec:
+    containers:
+    - args:
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: I believe you need `make` as the first entry in `command`

Comment (m1kola) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: @stevekuznetsov sorry, I'm not sure that I understand what you mean. Should it be defined differently from the `periodic-ci-azure-vmimage` job above and other jobs?

**Pull request merged**